code/game/objects/effects/decals/decal.dm

```diff
@@ -8,7 +8,8 @@
 /obj/effect/decal/Initialize()
 	. = ..()
 	if(turf_loc_check && (!isturf(loc) || NeverShouldHaveComeHere(loc)))
-		return INITIALIZE_HINT_QDEL
+		if (!istype(src, /obj/effect/decal/cleanable/crayon) && isclosedturf(loc)) //graffiti on walls

```

Comment:
Cmon you have a function that does this right there and you stick in istype.

Reply:
You won't catch me calling myself a good coder

Not sure what you're referring to though, it should only affect crayon drawings

Reply:
Just to be clear I'd still be using istype, just within NeverShouldHaveComeHere?

I'm wondering if it wouldn't make more sense to allow all decals on closed turfs after all, they layer under them anyway so I can't think of a good reason not to.

Would still have the issue on how to layer graffiti on top of them but that was going to happen either way

Reply:
If you mean istypes from base nevershouldhavecomehere then yes, it would be just the core one without the closed turf check. I'm not sure about allowing this for all decals, this might have some side effects.

Reply:
I think I got it, thank you so much for the help